### PR TITLE
FR: Bonus Quests And EP1 Client Orders

### PR DESCRIPTION
**Pull request by UnLaitQuiDanse**
The following files have been translated to french:
- CLIENT ORDERS -
co01010001.csv
co01010020.csv
co01010030.csv

- BONUS QUESTS -
quest_00150000.csv
quest_00150010.csv 
quest_00150030.csv 
quest_00150040.csv
quest_00150060.csv 
quest_00150070.csv

### CODE REVIEW

File: Orders/Client/EP1/co01010001.csv [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Title#0,"""Level Cap Trial II """
```
Comment (alama) [inline Orders/Client/EP1/co01010001.csv]: Please do not CHANGE this text or add spaces to bypass the EN string check

File: Orders/Client/EP1/co01010001.csv [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+Explanation#0,"""Il s'agit d'un essai pour demander l'obtention du<br>level cap prescrit par la reglementation ARKS.<br>Si vous reussissez cette mission<br>vos abilites seront reconnues, et vous<br>will be able to raise your level further.<br>pourrez augmenter votre niveau.<br>Ayez confiance en vous, et<br>n'ayez pas peur de vous mettre au defi.<br><br><yellow>※Les objets sont droppes par Nv.21+ Natifs, <br>Dragonkin et Mechs.<c>"""
+Errand#0,"""Level Cap License II~VI """
```
Comment (alama) [inline Orders/Client/EP1/co01010001.csv]: AND this

File: Quests/Bonus Quests/quest_00150000.csv [hunk 3]
```diff
@@ -0,0 +1,3 @@
+Title#0,"""Mission Speciale: Tokyo [Gold]"""
+Explanation#0,"""Sur la planete Terre, nous sommes parvenus<br>a contenir certains ennemis<br>dans une section de Tokyo.<br>De plus, les forces ennemies<br>se sont beaucoup affaiblies.<br>Ne ratez pas cette chance. Rendez-vous vite dans<br>cette zone, et battez les ennemis!<br>―――――――――――――――――――<br>Start: Utilisez Clé Bonus Tokyo [Gold]<br>Objectif: Battez tous les ennemis avant la fin du temps."""
+SimpleExplanation#0,"""Battez tous les ennemis !"""
```
Comment (alama) [inline Quests/Bonus Quests/quest_00150000.csv]: Should there be a space between `ennemis` and `!`?

File: Quests/Bonus Quests/quest_00150010.csv [hunk 4]
```diff
@@ -0,0 +1,3 @@
+Title#0,"""Mission Speciale: Tokyo [Silver]"""
+Explanation#0,"""Sur la planete Terre, nous sommes parvenus<br>a contenir certains ennemis<br>dans une section de Tokyo.<br>De plus, les forces ennemies<br>se sont beaucoup affaiblies.<br>Ne ratez pas cette chance. Rendez-vous vite dans<br>cette zone, et battez les ennemis!<br>―――――――――――――――――――<br>Start: Utilisez Clé Bonus Tokyo [Silver]<br>Objectif: Battez tous les ennemis avant la fin du temps."""
+SimpleExplanation#0,"""Battez tous les ennemis !"""
```
Comment (alama) [inline Quests/Bonus Quests/quest_00150010.csv]: Same here

File: Quests/Bonus Quests/quest_00150000.csv (hunk 3 above)
Comment (UnLaitQuiDanse) [reply Quests/Bonus Quests/quest_00150000.csv]: corrected

File: Orders/Client/EP1/co01010001.csv (hunk 2 above)
Comment (UnLaitQuiDanse) [reply Orders/Client/EP1/co01010001.csv]: Corrected

File: Quests/Bonus Quests/quest_00150000.csv (hunk 3 above)
Comment (UnLaitQuiDanse) [reply Quests/Bonus Quests/quest_00150000.csv]: Not necessary, I change that

File: Orders/Client/EP1/co01010001.csv (hunk 1 above)
Comment (UnLaitQuiDanse) [reply Orders/Client/EP1/co01010001.csv]: Okay, but how can I avoid CircleCI's error in this case?

File: Orders/Client/EP1/co01010001.csv [hunk 5]
```diff
@@ -0,0 +1,3 @@
+Title#0,"""Level Cap Trial II"""
```
Comment (alama) [inline Orders/Client/EP1/co01010001.csv]: just changed 'Level Cap Trial II' back to `レベル制限解除試練・Ⅱ`

File: Orders/Client/EP1/co01010001.csv [hunk 6]
```diff
@@ -0,0 +1,3 @@
...
+Explanation#0,"""Il s'agit d'un essai pour demander l'obtention du<br>level cap prescrit par la reglementation ARKS.<br>Si vous reussissez cette mission<br>vos abilites seront reconnues, et vous<br>will be able to raise your level further.<br>pourrez augmenter votre niveau.<br>Ayez confiance en vous, et<br>n'ayez pas peur de vous mettre au defi.<br><br><yellow>※Les objets sont droppes par Nv.21+ Natifs, <br>Dragonkin et Mechs.<c>"""
+Errand#0,"""Level Cap License II~VI"""
```
Comment (alama) [inline Orders/Client/EP1/co01010001.csv]: and this back to `レベル制限解除許可証Ⅱ～Ⅵ`

File: Files/quest_00150030.csv [hunk 7]
```diff
@@ -1,3 +1,3 @@
-Title#0,"""特別任務：禍津【銀】"""
-Explanation#0,"""惑星ハルコタンの『禍津』、およびそれの<br>影響で出現した黒の民を、「唱結界」内に<br>閉じ込め、弱体化させることに成功した。<br>戦闘可能なアークスは、現地へと急行し<br>『禍津』、および黒の民を撃退せよ。<br>A.I.Sの出撃準備も整っているため<br>A.I.Sを利用した早期撃退を推奨する。<br>――――――――――――――――――――<br>開始条件：ボーナスキー禍津【銀】消費<br>敵の撃退または制限時間の経過でクリア"""
-SimpleExplanation#0,"""敵をすべて撃退せよ！"""
+Title#0,"""Mission Speciale: Magatsu [Argent]"""
```
Comment (alama) [inline Files/quest_00150030.csv]: You should had been able to MOVE the file at the same time of editing it

**Comment by UnLaitQuiDanse** [comment]
All Bonus Quests have been translated to French.

**Comment by alama** [comment]
If this is done, you COULD remove the WIP tag

**Pull request merged**